release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -4254,8 +4254,42 @@ You can view the container images in this release by running the following comma
 $ oc adm release info 4.12.41 --pullspecs
 ----
 
-
 [id="ocp-4-12-41-updating"]
 ==== Updating
 
 To update an existing {product-title} 4.12 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI].
+
+[id="ocp-4-12-42"]
+=== RHSA-2023:6276 - {product-title} 4.12.42 bug fix and security update
+
+Issued: 2023-11-08
+
+{product-title} release 4.12.42, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2023:6276[RHSA-2023:6276] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2023:6278[RHSA-2023:6278] advisory.
```

Comment:
Checking that this is expected behavior - the `6276` errata resolves, but the `6278` does not. Should they both be the same?

Reply:
Typo on my part. RHBA not RHSA